zenoblend/main.cpp

```diff
@@ -292,25 +292,65 @@ PYBIND11_MODULE(pylib_zenoblend, m) {
     });
 
     m.def("meshGetLoopsCount", []
-            ( uintptr_t meshPtr
-            ) -> size_t
+        ( uintptr_t meshPtr
+        ) -> size_t
     {
         auto mesh = reinterpret_cast<zeno::BlenderMesh *>(meshPtr);
         return mesh->loop.size();
     });
 
     m.def("meshGetLoops", []
-            ( uintptr_t meshPtr
-            , uintptr_t loopPtr
-            , size_t loopCount
-            ) -> void
+        ( uintptr_t meshPtr
+        , uintptr_t loopPtr
+        , size_t loopCount
+        , uintptr_t loopColorPtr
+        ) -> void
     {
         auto mesh = reinterpret_cast<zeno::BlenderMesh *>(meshPtr);
         auto loop = reinterpret_cast<MLoop *>(loopPtr);
+
         for (int i = 0; i < loopCount; i++) {
             loop[i].v = mesh->loop[i];
             loop[i].e = 0;
         }
+
+        if(loopColorPtr) {
+            auto iter = mesh->vert.attrs.find("clr");
```

Comment:
"clr" can be either `float3` or `float`, which then will be converted into blender mesh vertex color.